src/usr/local/buildpack/tools/npm.sh

```diff
@@ -10,9 +10,13 @@ fi
 
 npm install -g npm@${TOOL_VERSION}
 
+hash -d npm
+
 npm --version
 
 # Clean download cache
 npm cache clean --force
 # Clean node-gyp cache
-rm -rf /root/.cache
+if [[ $EUID -eq 0 ]]; then
+  rm -rf /root/.cache
```

Comment:
Would non-user cache be in `/home/user/.cache`?

Reply:
maybe, not sure